CONTRIBUTING.MD

```diff
@@ -1,117 +1,54 @@
 # How to contribute
 
-## Developing Locally
-
-### Developing Packages that depend on one another at the same time
-
-1. Run `yarn` at the root of the repo.
-
-2. Run `yarn lerna link --force-local` at the root of the repo.
-
-3. Run `yarn build` at the root of the repo.
-
-4. Run `yarn dev` at each of the packages that depends on the other.
-
-> You can run a package individually: `yarn workspace {packageName} develop`
-
-### Developing packages while also testing on a store
-
-1. Repeat the steps from the above section
-
-2. Run `yarn link` on the package that you are developing
-
-3. Run `yarn` at the store's repo
-
-4. If the store's repo already have the package that you are developing as a dependency, just run `yarn link {packageName}`
+1. Fork this repository.
 
-5. If your package is not an npm dependency yet, you can develop locally by referring to your local version of the package on gatsby config.
+   > Click the "Fork" button in the upper right corner to create a copy (fork) of the repository under your GitHub account.
 
-Example:
+2. Clone your forked `faststore` repository.
+3. Create a new branch for the changes you are working on.
 
-```ts
-{
-  resolve: '/Users/jmgris/work/faststore/packages/gatsby-plugin-pixel-facebook'
-}
-```
+## Developing Locally
 
-### Creating Components on the `faststore/ui`
+### Developing Packages
 
-You can generate the boilerplate files for your new component using the following command:
+1. Run `yarn` at the root of the repo.
 
-- run `yarn generate-ui-component`
+2. Run `yarn build` at the root of the repo.
 
-```bash
-# What is your component name?
+3. Run `yarn dev` at each of the packages that depends on the other.
+   > You can run a package individually: `yarn workspace {packageName} dev`
```

Comment:
We're using turborepo, so we need to encourage the people to use it too.

Here is an example:

```cmd
yarn turbo run dev:server --filter=@faststore/api
```